nvflare/app_common/aggregators/accumulate_model_aggregator.py

```diff
@@ -67,7 +67,7 @@ def accept(self, shareable: Shareable, fl_ctx: FLContext) -> bool:
         except:
             self.log_exception(fl_ctx, "shareable data is not a valid DXO")
             return False
-        assert isinstance(dxo, DXO)
+        assert isinstance(dxo, DXO), "dxo must be DXO but got {}".format(type(dxo))
```

Comment:
Same here, we need to think if we need this check or not.

If we need: suggest change to if instance raise
If we don't: remove this whole assert statement